numalogic/udfs/_config.py

```diff
@@ -44,6 +44,7 @@ class MLPipelineConf:
 
     pipeline_id: str = "default"
     metrics: list[str] = field(default_factory=list)
+    combine_metrics: bool = False
```

Comment:
what is this for?

Reply:
for instance: [failed, degraded]

now all the metrics, pipeline payload are looking for 1 feature after the transform. Hence, this was needed.

Reply:
This get's checked in preprocess step and based on the value we combine the metrics name too